src/commands/dev/index.js

```diff
@@ -45,14 +45,14 @@ function addonUrl(addonUrls, req) {
   return addonUrl ? `${addonUrl}${m[2]}` : null
 }
 
-async function isStatic(pathname, publicFolder) {
+async function getStatic(pathname, publicFolder) {
   const alternatives = alternativePathsFor(pathname).map(p => path.resolve(publicFolder, p.substr(1)))
 
   for (const i in alternatives) {
     const p = alternatives[i]
     try {
       const pathStats = await stat(p)
-      if (pathStats.isFile()) return true
+      if (pathStats.isFile()) return '/'+path.relative(publicFolder, p)
```

Comment:
Instead of just checking for static file, we're retuning the path to file that exists, so that, we can use it `serveRedirect` function.